### ISSUE TITLE
Alerts for 30s metrics

### ISSUE THREAD

**Issue by leehinman** [elastic/elastic-agent#3853]
**Describe the enhancement:**

Alerts for 30s metrics

**Describe a specific use case for the enhancement or feature:**

We should have alerts for the 30s metrics for common performance problems.  For example if the queue is always full this should trigger an alert that performance tuning is necessary

**What is the definition of done?**

- [ ] Alert for constantly full queue
- [ ] Alert for retries
- [ ] Additional alerts
- [ ] Alerts can be loaded into Kibana

**Comment by cmacknz** [comment]
Are these metrics also collected by the monitoring metricbeat we start? It is using the stack monitoring Beat module. If they aren't, should we add them there?

If they are, then we have them at a higher frequency in the `metrics-elastic_agent.*` data streams. They are just duplicated in the logs so that we always have access to them in the case that the user has disabled metric monitoring or is using a standalone agent without Fleet. 

https://github.com/elastic/elastic-agent/blob/55b01c668c274141909b61baeb397bbd457b809a/internal/pkg/agent/application/monitoring/v1_monitor.go#L531-L625

**Comment by leehinman** [comment]
> Are these metrics also collected by the monitoring metricbeat we start? It is using the stack monitoring Beat module. If they aren't, should we add them there?

Yes, same metrics.  And yes, these alerts should get added to stack monitoring.  But for this first pass of development I want to keep them separate so we can use them for debugging support cases and figure out what good limits are for the alerts and what kind of alerts are useful.
